public/grpc-client.js

```diff
@@ -72,6 +73,11 @@ module.exports.init = async function({
   });
 
   ipcMain.on('lndInit', event => {
+    const macaroonCreds = getMacaroonCreds(lndSettingsDir, network);
+    credentials = grpc.credentials.combineChannelCredentials(
+      credentials,
+      macaroonCreds
+    );
     lnd = new lnrpc.Lightning(`localhost:${lndPort}`, credentials);
```

Comment:
So we don't need to pass the metadata in each api call because we inject the credentials here?

Reply:
Yep, thought it simplified the code a bit to not have to remember to pass in `metadata` each time.

Reply:
If we stuck with passing in metadata each time, we'd also have to get the macaroons to `action/grpc` and pass them into the `Duplex` here: https://github.com/lightninglabs/lightning-app/blob/master/src/action/grpc.js#L96

Reply:
Ok great 👍